### PR TITLE
Update LIT1982Mashaf.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
In order to avoid doublets, I supplied incipit and abstract to the already existing LIT1982Mashaf
I also changed the transcription of the title to Maṣḥafa ̣Ṣ́aḥāy. When this record is merged, the newly created ID 6818 is to be deleted.

### CODE REVIEW

File: 1001-2000/LIT1982Mashaf.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Mäṣḥafä ḍäḥay</title>
+            <title xml:id="t1">Maṣḥafa ̣Ṣ́aḥāy</title>
```
Comment (eu-genia) [inline 1001-2000/LIT1982Mashaf.xml:7]: title in gez
title normalized
title translated

File: 1001-2000/LIT1982Mashaf.xml [hunk 2]
```diff
@@ -33,17 +33,39 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-         <ab/>
-      </body>
+         <body>
+            <div xml:lang="gez" type="edition">
+               <note>The incipit is taken from <ref type="mss" corresp="BMLacq38#ms_i1"/></note>
+               <div type="textpart" subtype="incipit">
+                  <ab xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ። ወእምዝ፡ ንዌጥን፡ በረድኤተ፡ እግዚአብሔር፡ እንዘ፡ ንጽሕፍ፡ ዘንተ፡ መጽሐፈ፡ 
+                     ዘይሰመይ፡ መጽሐፈ፡ ጸሐይ። ወሶበ፡ ይበውእ፡ ውስተ፡ ልበ፡ ሰብእ፡ ያበርህ፡ ከመ፡ ፀሐይ። ይቤ፡ ኢሳይያስ፡ ወልደ፡ ትምህርቶሙ፡ ለቢዘናት። ቀዳሚ፡ ትንቢት፡ እምደብረ፡ 
+                     አራራት፡ ወአፈዋተ፡ ከርቤ፡ እም፡ አውግረ፡ ስኂን፡ ልዑላት፡ ዘውእቶሙ፡ ሐዋርያት። </ab>
+               </div>
```
Comment (eu-genia) [inline 1001-2000/LIT1982Mashaf.xml:67]: copy explicit already present in the issue as well
Explicit: እለኒ፡ ተኃቅፉ፡ እስራኤል፡ ቀዳሚ፡ ወይእዜሰ፡ ወጽኡ፡ እም፡ ታኅተ፡ ክንፊሁ፡ ለልዑል። እለኒ፡ ተኃጕሉ፡ እስራኤል፡ ወእለኒ፡ ተኃሠዩ፡ አሕዛብ። እለሰ፡ ተዓቀቡ፡ ቀዳሚ፡ እስራኤል፡ ወይእዜሰ፡ ወጽኡ፡ እምይብተራሁ፡ ወተማሠጥዎሙ፡ ፀላእቶሙ። ተመንደብኡ፡ በመርኃ፡ ብዙኅ።

File: 1001-2000/LIT1982Mashaf.xml (hunk 1 above)
Comment (eu-genia) [reply 1001-2000/LIT1982Mashaf.xml:7]: check https://betamasaheft.eu/Guidelines/?id=transliteration-principles 
ḍ is correct

**Review by eu-genia — CHANGES_REQUESTED** [review]
https://github.com/BetaMasaheft/Documentation/issues/2336

File: 1001-2000/LIT1982Mashaf.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply 1001-2000/LIT1982Mashaf.xml:7]: A encoded the Gəʿəz-title with xml:lang="gez"; the transliteration I encoded as type="normalized" (but not: type="transliterated", what was also offered by Oxygen); The English title I encoded with xml:lang="en"
```
<titleStmt>
            <title xml:lang="gez" xml:id="t1">መጽሐፈ፡ ፀሓይ፡ </title>
            <title type="normalized" xml:id="t2">Maṣḥafa ḍaḥāy</title>
            <title xml:lang="en" xml:id="t3">Book of the Sun</title>
            <editor role="generalEditor" key="AB"/>
            <funder>Akademie der Wissenschaften in Hamburg</funder>
         </titleStmt>
```
Is this all correct?

File: 1001-2000/LIT1982Mashaf.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply 1001-2000/LIT1982Mashaf.xml:69]: Done.

File: 1001-2000/LIT1982Mashaf.xml [hunk 3]
```diff
@@ -4,7 +4,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Mäṣḥafä ḍäḥay</title>
+            <title xml:lang="gez" xml:id="t1">መጽሐፈ፡ ፀሓይ፡ </title>
+            <title type="normalized" xml:id="t2">Maṣḥafa ḍaḥāy</title>
```
Comment (eu-genia) [inline 1001-2000/LIT1982Mashaf.xml:8]: not xml:id="t2" and "t3" but `corresp="#t1"`, it is still title number 1 with transcription and translation

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: 1001-2000/LIT1982Mashaf.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply 1001-2000/LIT1982Mashaf.xml]: Yes, I have just seen it in the guidelines that you linked to me a minute ago.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**